app/gateway/changelog.md

```diff
@@ -11,6 +11,26 @@ For Kong Gateway OSS, view the [OSS changelog on GitHub](https://github.com/Kong
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.7.1.2
+**Release Date** 2024/07/05
+
+### Features
+
+#### Plugins
+
+* [**AWS Lambda**](/hub/kong-inc/aws-lambda) (`aws-lambda`)
```

Comment:
@GGabriele Is this one?

Reply:
[The one](https://github.com/Kong/kong-ee/pull/9425/files#diff-ba89bd07f45f08f8b16d4e53b2ee3b183d98686144e508969c432e1ed7fccf1a) I'm referring to is about `request-validator`, not `aws-lambda`

Reply:
@VicYP @GGabriele What is this in reference to? I don't see anything about request-validator in the EE changelogs.